docs/autodoc_submodules.py

```diff
@@ -51,23 +66,49 @@ def op_autodoc(out_filename):
         if module in exclude_ops_members:
             excluded = exclude_ops_members[module]
             s += "   :exclude-members: {}\n".format(", ".join(excluded))
-        s += "\n"
+        s += "\n\n"
     with open(out_filename, 'w') as f:
         f.write(s)
 
-def fn_autodoc(out_filename):
+def get_module_references(module_name, references):
+    """Generate section with references for given module"""
+    module_name = module_name[12:] # remove nvidia.dali prefix
+    result = ""
+    if module_name in references:
+        result += ".. seealso::\n"
+        for desc, url in references[module_name]:
+            result += f"   * `{desc} <./{url}>`_\n"
+    return result
+
+def get_operator_references(op_name, references):
+    """Generate section with references for given operator"""
+    op_name = op_name[12:] # remove nvidia.dali prefix
+    result = ""
+    if op_name in references:
+        result += ".. seealso::\n"
+        for desc, url in references[op_name]:
+            result += f"   * `{desc} <./{url}>`_\n"
+    return result
+
+
+def fn_autodoc(out_filename, references):
     s = ""
-    for module in get_modules(fn_modules):
+    all_modules = get_modules(fn_modules)
+    for module in all_modules:
+        dali_module = sys.modules[module]
+        # Take all public members of given module
+        funs_in_module = get_functions(dali_module)
         s += module + "\n"
         s += "~" * len(module) + "\n"
         if module in mod_aditional_doc:
             s += mod_aditional_doc[module] + "\n" + "\n"
-        s += ".. automodule:: {}\n".format(module)
-        s += "   :members:\n"
-        s += "   :undoc-members:\n"
-        if module in exclude_fn_members:
-            excluded = exclude_fn_members[module]
-            s += "   :exclude-members: {}\n".format(", ".join(excluded))
+        s += get_module_references(module, references)
```

Comment:
Shouldn't you update op_autodoc as well?

Reply:
I'm not sure. Should we work on the deprecated API? It was my intention to not modify it.

Reply:
But you need to do a dedicated discovery of classes, there might be some gotchas, and the examples no longer contain any references to the old API. 
I am not in favor of utilizing additional resources for the API that we want to discourage people from using.

Reply:
I think we need to discuss the `ops` style API docs offline.